test/decorator-annotator_test.ts

```diff
@@ -28,68 +30,82 @@ function sources(sourceText: string): Map<string, string> {
   return sources;
 }
 
-function verifyCompiles(sourceText: string) {
-  // This throws an exception on error.
-  testSupport.createProgram(sources(sourceText));
-}
+describe('decorator-annotator', () => {
+  function translate(sourceText: string, allowErrors = false) {
+    const {host, program} = createProgramAndHost(sources(sourceText), compilerOptions);
+    if (!allowErrors) {
+      expect(ts.getPreEmitDiagnostics(program))
```

Comment:
can you follow the `to.be.empty` pattern we use below? I might be misremembering, but I think I explicitly removed this pattern before because it doesn't actually assert and fail. In any case, `to.be.empty` is more idiomatic.

Reply:
Done.